oem/ibm/libpldmresponder/file_io_type_dump.cpp

```diff
@@ -32,19 +32,38 @@ static constexpr auto dumpObjPath = "/xyz/openbmc_project/dump/system";
 static constexpr auto systemDumpEntry = "xyz.openbmc_project.Dump.Entry.System";
 static constexpr auto resDumpEntry = "com.ibm.Dump.Entry.Resource";
 static constexpr auto dumpEntryObjPath =
-    "/xyz/openbmc_project/dump/system/entry/";
+    "/xyz/openbmc_project/dump/system/entry";
 static constexpr auto bmcDumpObjPath = "/xyz/openbmc_project/dump/bmc/entry";
 
 // Resource dump file path to be deleted once hyperviosr validates the input
 // parameters. Need to re-look in to this name when we support multiple
 // resource dumps.
-static constexpr auto resDumpDirPath = "/var/lib/pldm/resourcedump/1";
 
 int DumpHandler::fd = -1;
 namespace fs = std::filesystem;
 
+uint32_t getDumpIdPrefix(uint16_t dumpType)
+{
+    switch (dumpType)
+    {
+        case PLDM_FILE_TYPE_HARDWARE_DUMP:
+            return 0x00000000;
+        case PLDM_FILE_TYPE_HOSTBOOT_DUMP:
+            return 0x20000000;
+        case PLDM_FILE_TYPE_SBE_DUMP:
+            return 0x30000000;
+        case PLDM_FILE_TYPE_RESOURCE_DUMP_PARMS:
+            return 0xB0000000;
+        default:
+            error("unsupported {TYPE}", "TYPE", dumpType);
+    }
+    return 0xFF;
+}
+
 std::string DumpHandler::findDumpObjPath(uint32_t fileHandle)
 {
+    info("FileHandle in findDumpObjPath is {FILEHANDLE}", "FILEHANDLE",
```

Comment:
Can we remove this debug trace ?

Reply:
Keeping this trace until the new dump support is stable from all the components.